tests/src/test/scala/org/http4s/parser/QueryParserSpec.scala

```diff
@@ -13,6 +11,9 @@ class QueryParserSpec extends Http4sSpec {
     QueryParser.parseQueryString(str)
 
   "The QueryParser" should {
+    import org.specs2.matcher._
+    def be_\/-(q: Query): RightDisjunctionCheckedMatcher[Query] =
```

Comment:
Without this, we get compilation errors like this, but only in Scala 2.10:

```
[error] /Users/bakro29/src/http4s/tests/src/test/scala/org/http4s/parser/QueryParserSpec.scala:18: could not find implicit value for evidence parameter of type org.specs2.execute.AsResult[(String, Option[String])] [error] parseQueryString("key=value") must be_\/-(Query("key" -> Some("value")))
```

I'm not sure why.  I associate this error with not returning an assertion from a spec.  @etorreborre or any other specs2 experts have an idea?

I can live with the workaround for now: it's confined to a test, and we can clean it up after the 0.15 release.